### PR TITLE
Add notes from 'Tools, Tips, and Tricks' session

### PR DESCRIPTION
**Pull request by bolducp**
My notes from the 'Tools, Tips, and Tricks' session, written in markdown.

### CODE REVIEW

File: tools_tips_and_tricks.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+### TOOLS, TIPS, & TRICKS SESSION
+
+* yaml parser
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: http://yaml-online-parser.appspot.com/

File: tools_tips_and_tricks.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* json validator
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: https://jsonformatter.curiousconcept.com/

File: tools_tips_and_tricks.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* rubular (visualizes the regular expression given in Ruby against a text)
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: http://rubular.com/

File: tools_tips_and_tricks.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+* dirtymock.com (can clean up html, javascript, etc.) 
+* Postman chrome extension (GUI for curl; raw http requests)
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: https://www.getpostman.com/

File: tools_tips_and_tricks.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* hurl.it (similar to Postman)
+* Ngrok 
+* JSON View Chrome extension (for viewing JSON in the browser)
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc

File: tools_tips_and_tricks.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+* JQ (for working with JSON; defines a query language for parsing the JSON data structure)
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: https://stedolan.github.io/jq/

File: tools_tips_and_tricks.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+* dirtymock.com (can clean up html, javascript, etc.) 
+* Postman chrome extension (GUI for curl; raw http requests)
+* hurl.it (similar to Postman)
+* Ngrok 
+* JSON View Chrome extension (for viewing JSON in the browser)
+* JQ (for working with JSON; defines a query language for parsing the JSON data structure) 
+* Capture (for taking and sharing screenshots)
+* Giphy Capture (or Grabber?)  (for making a gif of what’s happening on the screen)
+* Spectacle
+* Dash (utility that gives the offline documentation; integrates with Off road)
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: https://kapeli.com/dash

File: tools_tips_and_tricks.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+* Heroku feature— “review app” 
+* isyourmonitoron.com
+* graphisrepl.com  (graphus is a tool like markdown)
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: http://www.graphvizrepl.com/

File: tools_tips_and_tricks.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+* Giphy Capture (or Grabber?)  (for making a gif of what’s happening on the screen)
+* Spectacle
+* Dash (utility that gives the offline documentation; integrates with Off road)
+* Heroku feature— “review app” 
+* isyourmonitoron.com
+* graphisrepl.com  (graphus is a tool like markdown) 
+* pullrequestroulette.com
+* Vim extensions: Janus (set of customizations for Ruby programmers) 
+	www.github.com/carlhuda/janus
+* Vim: remapping the CAPSLOCK key to Ctrl
```
Comment (BanzaiMan) [inline tools_tips_and_tricks.md]: You can do this universally on OS X/macOS: System Preferences → Keyboard → Keyboard (tab) → Modifier Keys (lower right button).

Comment (bolducp) [reply tools_tips_and_tricks.md]: @BanzaiMan -- awesome! thanks for all of the corrections and additions. I'll add these to the README tonight and update my PR.

**Review by valeriecodes — APPROVED** [review]

**Pull request merged**